app/src/main/java/org/oppia/app/home/HomeActivity.kt

```diff
@@ -24,4 +28,17 @@ class HomeActivity : InjectableAppCompatActivity(), RouteToExplorationListener,
   override fun routeToTopic(topicId: String) {
     startActivity(TopicActivity.createTopicActivityIntent(this, topicId))
   }
+
+  override fun onBackPressed() {
```

Comment:
Please also add a test for this behavior.

Reply:
Updated tests.